php-apache/7.0/usr/local/share/env/default_env_variables

```diff
@@ -12,3 +12,9 @@ export TIDEWAYS_API_KEY=${TIDEWAYS_API_KEY:-}
 export TIDEWAYS_CONNECTION=${TIDEWAYS_CONNECTION:-tcp://tideways:9135}
 
 export WORK_DIRECTORY=${WORK_DIRECTORY:-/app}
+
+export BASIC_AUTH_ENABLED=${BASIC_AUTH_ENABLED:-}
+export BASIC_AUTH_REALM=${BASIC_AUTH_REALM:-Protected System}
+export BASIC_AUTH_FILE=${BASIC_AUTH_FILE:-/etc/apache2/htpasswd}
+export BASIC_AUTH_TYPE=${BASIC_AUTH_TYPE:-Basic}
```

Comment:
It's an easy change for the naming of variables - and better now before anything implements it. Pushing in a minute!